docs/Controllers.md

```diff
@@ -0,0 +1,120 @@
+# Controllers
+
+_(c) AMWA 2022, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
+
+## Informative
+
+A Controller is Client software that interacts with the NMOS APIs to discover, connect and manage devices within a networked media system.
```

Comment:
Hmm, Client is capitalized here, but the term is only used lowercase in IS-04 and IS-05 currently. It is defined in BCP-003-01, but if we want to use it as a defined term that needs to be repeated or maybe referenced, doesn't it?

https://specs.amwa.tv/bcp-003-01/releases/v1.0.0/docs/1.0._Secure_Communication.html#client

Reply:
ARG - mid term plan, think about normative NMOS Core (with defined terms and general behaviours) or at least central definition of terms, but in short term add relevant definitions to IS-04 and IS-05.

Reply:
Or point to (informative) [NMOS Glossary](https://specs.amwa.tv/nmos/main/docs/Glossary.html) from IS-04 and IS-05 and add definitions for Client and Controller there?

Reply:
Or point to (informative) [NMOS Glossary](https://specs.amwa.tv/nmos/main/docs/Glossary.html) from IS-04 and IS-05 and add definitions for Client and Controller there?

Reply:
Resolved, Glossary link added in Overview